docs/source/development/index.rst

```diff
@@ -59,8 +59,8 @@ releases.
 
 .. code::
 
-    $ python setup.py sdist --formats=zip,gztar
-    $ python setup.py bdist_wheel
+    python setup.py sdist --formats=zip,gztar
+    python setup.py bdist_wheel --universal
```

Comment:
I think the universal option can be set in `setup.cfg` if that's preferable to specifying it manually. But for now, this is correct.